sitemap/generator.go

```diff
@@ -19,6 +19,7 @@ type Files map[config.Language]string
 type FileStore interface {
 	SaveFile(name string, body io.Reader) error
 	GetFile(name string) (body io.ReadCloser, err error)
+	SaveFiles(paths []string) error
```

Comment:
I think it would be better to keep this interface lean and move the looping functionality up a level